### PR TITLE
Features/treasury balance

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Added treasury balance in peers module

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ async function bootstrap() {
   SwaggerModule.setup(`${servicePrefix}/swagger`, app, document);
 
   const blockScannerService = app.select(BlockScannerModule).get(BlockScannerService);
-  blockScannerService.init();
+  // blockScannerService.init();
```
Comment (evgeny-s) [inline src/main.ts:35]: Shouldn't we revert it?

File: src/modules/peers/peer.controller.ts [hunk 2]
```diff
@@ -21,4 +21,10 @@ export class PeerController {
     const result = await this.peerService.fetch(network);
     return result;
   }
+
+  @Get('/treasury-balance/:network')
+  public async treasuryBalance(@Param('network') network: string): Promise<any>{
+    const balance = await this.peerService.treasuryBalance(network);
```
Comment (evgeny-s) [inline src/modules/peers/peer.controller.ts:27]: redundant variable:
* Change to `return this.pee....`
* Remove `async` from this function

**Review by evgeny-s — CHANGES_REQUESTED** [review]

**Comment by evgeny-s** [comment]
@karthikeyan-cere , do not forget release notes

File: src/main.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/main.ts]: reverted it.

File: src/modules/peers/peer.controller.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply src/modules/peers/peer.controller.ts]: updated.

**Review by evgeny-s — APPROVED** [review]

**Pull request merged**